ci/setup-linux.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -x
+
+apt-get update
+apt-get -y install build-essential pkg-config cmake openssl libssl-dev libssh2-1-dev libcurl4-gnutls-dev openssh-server
+
+if [ ! -d /var/run/sshd ]; then
+	sudo mkdir /var/run/sshd
```

Comment:
`sudo` is not necessary here, as we're already running with elevated privileges. You can also avoid the conditional by just using `mkdir -p`, which will not complain if the target directory already exists.